sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/models/TableErrorCode.java

```diff
@@ -8,43 +8,277 @@
+     */
+    public static final TableErrorCode METHOD_NOT_ALLOWED = fromString("MethodNotAllowedValue");
+
+    /**
+     * Static value {@code ContentLengthExceeded}.
+     */
+    public static final TableErrorCode CONTENT_LENGTH_EXCEEDED = fromString("ContentLengthExceededValue");
+
+    /**
+     * Static value {@code AccountIOPSLimitExceeded}.
+     */
+    public static final TableErrorCode ACCOUNT_IOPS_LIMIT_EXCEEDED = fromString("AccountIOPSLimitExceededValue");
+
+    /**
+     * Static value {@code CannotCreateTableWithIOPSGreaterThanMaxAllowedPerTable}.
+     */
+    public static final TableErrorCode CANNOT_CREATE_TABLE_WITH_IOPS_GREATER_THAN_MAX_ALLOWED_PER_TABLE =
+        fromString("CannotCreateTableWithIOPSGreaterThanMaxAllowedPerTableValue");
+
+    /**
+     * Static value {@code PerTableIOPSIncrementLimitReached}.
+     */
+    public static final TableErrorCode PER_TABLE_IOPS_INCREMENT_LIMIT_REACHED =
+        fromString("PerTableIOPSIncrementLimitReachedValue");
+
+    /**
+     * Static value {@code PerTableIOPSDecrementLimitReached}.
+     */
+    public static final TableErrorCode PER_TABLE_IOPS_DECREMENT_LIMIT_REACHED =
+        fromString("PerTableIOPSDecrementLimitReachedValue");
+
+    /**
+     * Static value {@code SettingIOPSForATableInProvisioningNotAllowed}.
+     */
+    public static final TableErrorCode SETTING_IOPS_FOR_A_TABLE_IN_PROVISIONING_NOT_ALLOWED =
+        fromString("SettingIOPSForATableInProvisioningNotAllowedValue");
+
+    /**
+     * Static value {@code PartitionKeyEqualityComparisonExpected}.
+     */
+    public static final TableErrorCode PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED =
+        fromString("PartitionKeyEqualityComparisonExpectedValue");
+
+    /**
+     * Static value {@code PartitionKeySpecifiedMoreThanOnce}.
+     */
+    public static final TableErrorCode PARTITION_KEY_SPECIFIED_MORE_THAN_ONCE =
+        fromString("PartitionKeySpecifiedMoreThanOnceValue");
+
+    /**
+     * Static value {@code InvalidInput}.
+     */
+    public static final TableErrorCode INVALID_INPUT = fromString("InvalidInputValue");
+
+    /**
+     * Static value {@code NotImplemented}.
+     */
+    public static final TableErrorCode NOT_IMPLEMENTED = fromString("NotImplementedValue");
+
+    /**
+     * Static value {@code OperationTimedOut}.
+     */
+    public static final TableErrorCode OPERATION_TIMED_OUT = fromString("OperationTimedOutValue");
+
+    /**
+     * Static value {@code OutOfRangeInput}.
+     */
+    public static final TableErrorCode OUT_OF_RANGE_INPUT = fromString("OutOfRangeInputValue");
+
+    /**
+     * Static value {@code Forbidden}.
+     */
+    public static final TableErrorCode FORBIDDEN = fromString("ForbiddenValue");
+
+    /**
+     * Static value {@code AuthorizationPermissionMismatch}.
+     */
+    public static final TableErrorCode AUTHORIZATION_PERMISSION_MISMATCH =
+        fromString("AuthorizationPermissionMismatch");
```

Comment:
This one also doesn't have "Value"

Reply:
None should have the value suffix, it was a mistake when copying the new entries from .NET

Reply:
Good catch!

Reply:
Good catch!